### PR TITLE
Fixes #33945 - add 'run a job' action to new host

### PR DESCRIPTION
**Pull request by amirfefer**
This adds a `Run a Job` action to the new host's kebab dropdown and also to the recent jobs card's kebab

### CODE REVIEW

**Comment by adamruzicka** [comment]
I'm getting this in the console, I guess that's relevant
image

File: .gitignore [hunk 1]
```diff
@@ -15,3 +15,4 @@ Gemfile.lock
 node_modules/
 package-lock.json
 coverage/
+jsconfig.json
```
Comment (MariaAga) [inline .gitignore:18]: This is not related to the pr, can it be done in a new one?

**Comment by MariaAga** [comment]
Can this pr add some test to the new action?

**Comment by amirfefer** [comment]
Thanks, @adamruzicka , @MariaAga 👍

**Comment by adamruzicka** [comment]
> Can this pr add some test to the new action?

@amirfefer How about this :point_up:? Apart from that LGTM

**Comment by amirfefer** [comment]
I'm closing this in favour https://github.com/theforeman/foreman_remote_execution/pull/679
